lib/dnsimple/client/certificates.rb

```diff
@@ -108,6 +108,167 @@ def certificate_private_key(account_id, domain_id, certificate_id, options = {})
+      #
+      # @see https://developer.dnsimple.com/v2/domains/certificates/#letsencrypt-purchase-renewal
+      #
+      # @param  account_id [Integer] the account ID
+      # @param  domain_id [#to_s] the domain ID or domain name
+      # @param  certificate_id [Integer] the certificate ID
+      # @param  attributes [Hash]
+      # @option attributes [TrueClass,FalseClass] :auto_renew enable certificate auto renew (optional)
+      # @param  options [Hash]
+      #
+      # @return [Dnsimple::Response<Dnsimple::Struct::CertificateRenewal>]
+      #
+      # @raise  [Dnsimple::NotFoundError]
+      # @raise  [Dnsimple::RequestError]
+      #
+      # @example Basic usage
+      #   reponse             = client.certificates.letsencrypt_purchase(1010, "example.com", 200)
+      #   certificate_renewal = response.data
+      #
+      #   certificate_renewal.id                 # => 999
+      #   certificate_renewal.old_certificate_id # => 200
+      #   certificate_renewal.new_certificate_id # => 200
+      #
+      # @example Auto renew
+      #   reponse             = client.certificates.letsencrypt_purchase(1010, "example.com", 200, auto_renew: true)
+      #   certificate_renewal = response.data
+      #
+      #   certificate_renewal.id                 # => 999
+      #   certificate_renewal.old_certificate_id # => 200
+      #   certificate_renewal.new_certificate_id # => 200
+      def letsencrypt_purchase_renewal(account_id, domain_id, certificate_id, attributes = {}, options = {})
+        response = client.post(Client.versioned("/%s/domains/%s/certificates/letsencrypt/%s/renewals" % [account_id, domain_id, certificate_id]), attributes, options)
+
+        Dnsimple::Response.new(response, Struct::CertificateRenewal.new(response["data"]))
+      end
+
+      # Issue a pending Let's Encrypt certificate renewal order.
+      #
+      # Note that the issuance process is async. A successful response means the issuance
+      # request has been successfully acknowledged and queued for processing.
+      #
+      # @see https://developer.dnsimple.com/v2/domains/certificates/#letsencrypt-issue-renewal
+      #
+      # @param  account_id [Integer] the account ID
+      # @param  domain_id [#to_s] the domain ID or domain name
+      # @param  certificate_id [Integer] the certificate ID
+      # @param  certificate_renewal_id [Integer] the certificate renewal ID
+      # @param  options [Hash]
+      #
+      # @return [Dnsimple::Response<Dnsimple::Struct::Certificate>]
+      #
+      # @raise  [Dnsimple::NotFoundError]
+      # @raise  [Dnsimple::RequestError]
+      #
+      # @example Basic usage
+      #   reponse     = client.certificates.letsencrypt_issue_renewal(1010, "example.com", 100, 999)
+      #   certificate = response.data
+      #
+      #   certificate.state # => "requesting"
+      def letsencrypt_issue_renewal(account_id, domain_id, certificate_id, certificate_renewal_id, options = {})
```

Comment:
What is the certificate ID here? Is it the old certificate? The new certificate?

Reply:
Old certificate.